flipperdevices/flipperzero-firmware
# [FL-3093, FL-3087] SubGhz: Fix Raw write, add short duration filter setting

# What's new

- SubGhz: Fix recording RAW files, sometimes could not start at a high level
- SubGhz: subghz_worker, add short duration filter setting

# Verification 

- Compile and upload
- Check `subzgh rx_raw`
- Check SubGhz RAW and Read

# Checklist (For Reviewer)
